### PR TITLE
Standardize license across source files

### PR DESCRIPTION
**Pull request by yangshun**
Some of the source files were still mentioning the BSD license. I changed them to CC-BY-4.0 and also made the year consistent as 2013 (some files were 2015).

### CODE REVIEW

File: src/utils/createLink.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
 /**
- * Copyright 2015-present, Facebook, Inc.
- * All rights reserved.
+ * Copyright (c) 2013-present, Facebook, Inc.
```
Comment (yangshun) [inline src/utils/createLink.js]: Made this line consistent with other files.

File: src/utils/createLink.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
  *
- * This source code is licensed under the BSD-style license found in the
```
Comment (yangshun) [inline src/utils/createLink.js]: Changed BSD to CC-BY-4.0

File: src/utils/sectionList.js [hunk 3]
```diff
@@ -1,13 +1,11 @@
 /**
- * Copyright 2015-present, Facebook, Inc.
- * All rights reserved.
+ * Copyright (c) 2013-present, Facebook, Inc.
```
Comment (yangshun) [inline src/utils/sectionList.js]: 2015 -> 2013

File: src/utils/sectionList.js [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
  *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
+ * This source code is licensed under the CC-BY-4.0 license found
```
Comment (yangshun) [inline src/utils/sectionList.js]: BSD -> CC-BY-4.0

**Comment by gaearon** [comment]
Thanks.

**Pull request merged**